backend/state.py

```diff
@@ -436,6 +436,8 @@ def step(self, actions, clock):
         if self.is_goal_reached():
             return self, True
         
+        # TODO: DEBUGGING
```

Comment:
remove comment if TODO already resolved, otherwise put the TODO in this format
`# TODO(your_username): This is an error message)`
and create a Git issue associated with it that should be mentioned in the PR